VMEncryption/main/CryptMountConfigUtil.py

```diff
@@ -0,0 +1,595 @@
+#!/usr/bin/env python
+#
+# VMEncryption extension
+#
+# Copyright 2019 Microsoft Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import os
+import os.path
+import re
+import shutil
+import uuid
+
+from CommandExecutor import CommandExecutor, ProcessCommunicator
+from Common import CryptItem, CommonVariables
+
+
+class CryptMountConfigUtil(object):
+    """A utility to modify the config files that mount or unlock encrypted disks"""
```

Comment:
> """A utility to modify the config files that mount or unlock encrypted disks""" [](start = 3, length = 80)

same. Add a 1-2 liner or more if you want defining what this and other functions actually do.